scripts/delete_db_and_setup.sh

```diff
@@ -1,6 +1,10 @@
-rm backend/run/db.sqlite3
-python3 backend/manage.py migrate
-echo "Creating account with username \"admin\". "
-echo "Please enter your desired email and password:"
-python3 backend/manage.py createsuperuser  --username admin
-python3 backend/manage.py loaddata backend/cms_export.json
+#!/bin/bash
+# To run script in Docker call for example "delete_db_and_setup.sh 'docker-compose exec backend'"
+EXEC_ENV="$*"
+
```

Comment:
It would be nice if you could run it from anywhere, but I don't know how to do that..
I tried this:
```suggestion
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" && cd $DIR/../
```

but then it would be a bit unexpected that you need to provide a docker-compose command taking into account the directory switch. On the other hand you wouldn't need to care about where you are when you call that script, it would always be `docker-compose -f docker-compose.yml -f docker-compose.prod.yml`

What do you think @Baschdl ?

Reply:
I think we would also have to specify the path of `backend/cms_export.json` dynamically. Maybe that's a bit too much engineering?

Reply:
Probably.. The suggestion I wrote switches to the project root dir reliably no matter from where you call the script (copied from StackOverflow, there is an error condition somewhere with symlinks that I forgot, but as you said... Didn't want to over-engineer. I wouldn't invest more time I think

Reply:
Not sure what to do here, I really like the idea but it may be a bit of an unexpected behavior for someone using it. We should probably maximize for the understandability for a new user

Reply:
Right now it needs to be called with `bash ../scripts/delete_and_setup_db.py` right?

Reply:
Yes, `cd backend && bash ../scripts/delete_and_setup_db.py` if you don't use docker

Reply:
With the proposed change it should work from everywhere.

Reply:
> would be a bit unexpected that you need to provide a docker-compose command taking into account the directory switch

@maltezacharias I'm not sure what you mean with this. It's always relative to `backend` if you use docker, right?

Reply:
My suggested line always checks where the script is located (the file itself) and then changes the workdir to the directory above. So you would always execute your commands from the repo root. That would mean you can either call it in one of the following ways:

```
# Using an absolute path from anywhere
/home/user/match4everything/scripts/delete_db_and_setup.sh  # for native django
/home/user/match4everything/scripts/delete_db_and_setup.sh docker-compose exec backend # for dev docker
/home/user/match4everything/scripts/delete_db_and_setup.sh docker-compose -f docker-compose.yml -f docker-compose.prod.yml exec backend # for dev docker

# From scripts dir
cd scripts
./delete_db_and_setup.sh  # for native django
./delete_db_and_setup.sh docker-compose exec backend # for dev docker
./delete_db_and_setup.sh docker-compose -f docker-compose.yml -f docker-compose.prod.yml exec backend # for dev docker

# from repo root
./scripts/delete_db_and_setup.sh  # for native django
./scripts/delete_db_and_setup.sh docker-compose exec backend # for dev docker
./scripts/delete_db_and_setup.sh docker-compose -f docker-compose.yml -f docker-compose.prod.yml exec backend # for dev docker
```

All of these should work equally (If I didn't make a typo)